packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -90,7 +136,8 @@ const SideNavExpandContainer = styled("nav", {
     display: "flex",
     opacity: isSideNavCollapsed ? 0 : 1,
     visibility: isSideNavCollapsed ? "hidden" : "visible",
-    width: isSideNavCollapsed ? "0" : expandedWidth,
+    width: isSideNavCollapsed ? 0 : expandedWidth,
+    minWidth: isSideNavCollapsed ? 0 : expandedWidth,
     transitionProperty: "opacity, width",
```

Comment:
Animating `width` isn't very performant. Is there a way we can do this with `transform: scaleX()` instead? Not necessarily a blocker but, we'll want to find some sort of solution that isn't width based animation

Reply:
yeah, I've removed this transition for now. It is not needed.